src/services/completions.ts

```diff
@@ -5767,20 +5771,39 @@ function isModuleSpecifierMissingOrEmpty(specifier: ModuleReference | Expression
     return !tryCast(isExternalModuleReference(specifier) ? specifier.expression : specifier, isStringLiteralLike)?.text;
 }
 
-function getVariableOrParameterDeclaration(contextToken: Node | undefined, location: Node) {
+function getClosestSymbolDeclaration(contextToken: Node | undefined, location: Node) {

```

Comment:
since `Parameter` has a specific meaning and commonly doesn't include `TypeParameter` I decided to rename this (as this function might return both of those types)